platform/lib/platform_web/live/media_live/edit_attribute.ex

```diff
@@ -3,6 +3,43 @@ defmodule PlatformWeb.MediaLive.EditAttribute do
   alias Platform.Material
   alias Material.Attribute
   alias Platform.Auditor
+  alias Platform.Uploads
+
+  def mount(socket) do
+    Temp.track!()
+
```

Comment:
Can we add a comment here that reminds folks that much of this logic also exists in `comment_box.ex` and so if they want to make changes, they should do so there as well?

(It'd be great to add a corresponding in `comment_box.ex` as well.)

```suggestion
# Note: the file upload logic is duplicated in `comment_box.ex`; if you change it, be sure to change `comment_box.ex` as well.
```